### ISSUE TITLE
Manifest error in project

### ISSUE THREAD

**Issue by DanAtkinson** [shanselman/babysmash#6]
Hi there!

Upon loading the project, the following error is thrown when building:

    Unable to find manifest signing certificate in the certificate store.

The project SignManifests value should be false in the public project unless you're happy to bundle the private.pfx with the project.

Kind regards, Dan

**Comment by DavidRieman** [comment]
Agreed:  In general, I feel it is a really good practice to have every code base build-ready by potential-new-contributors the moment they pull down the source.  Additional errors/barriers/preparation cut off potential interest/contributors before they even dive in for a look.  The code committed state of SignManifests should be false.  It should be the role of the release process to enable and perform signing (whether or not that process is automated).

**Comment by Razzeee** [comment]
Correct, but I don't know the release process of this project, so pardon my previous comment.
